src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/dashboardnotifications/defendant/ClaimantResponseDefendantNotificationHandler.java

```diff
@@ -90,6 +91,12 @@ public String getScenario(CaseData caseData) {
             }
         } else if (isClaimantRejectRepaymentPlan(caseData)) {
             return SCENARIO_AAA6_CLAIMANT_INTENT_REJECT_REPAYMENT_ORG_LTD_CO_DEFENDANT.getScenario();
+        } else if (
+            !featureToggleService.isJudgmentOnlineLive()
+                && !caseData.isApplicantLiP()
```

Comment:
Should it be caseData.isApplicantLiP()? You're checking if applicant is LR

Reply:
It is checking if applicant is LR or not. If applicant is LR the Applicant1Represented field can be null or set as YES